tests/Custom/ErrorHandlingTest.php

```diff
@@ -21,12 +25,11 @@ protected function setUp()
 
     public function testGetClient()
     {
-        $this->assertSame($this->client, $this->getDriver()->getClient());
+        $this->assertSame($this->client, $this->getDriver()->getBrowser());
     }
 
     /**
-     * @expectedException \Behat\Mink\Exception\DriverException
-     * @expectedExceptionMessage Unable to access the response before visiting a page
+     * @expectedException \Exception
```

Comment:
Why these changes?

Reply:
Exception change to `BadMethodCallException` on version 5, to keep test compatibily between each version i change for global exception.

Reply:
@FlorianPerrot, you can set expected exception dynamically in test method itself and not via DocBlock. In there you can check used Symfony version.

Reply:
I rollback and use `DriverException` ans keep compatibility with BadMethodCallException.
See https://github.com/minkphp/MinkBrowserKitDriver/pull/142/files#diff-5f6fd35d049866794211ba65ff1622ecR899